### ISSUE TITLE
how to change the size of label?

### ISSUE THREAD

**Issue by DJ-kim** [dcjones/Gadfly.jl#661]
![8a288ee6-2616-11e5-86b5-b0faa63fadab](https://cloud.githubusercontent.com/assets/13537753/9314465/19dfc3f8-4564-11e5-8986-561b7910716d.png)

As you see above, continuous color label too small(default size). Can I change the size of label?

**Comment by dcjones** [comment]
Yes, there are a few font size parameters in `Theme`. You can add `Theme(key_label_font_size=14pt)`, for example to make key labels larger. Other parameters: `minor_label_font_size`, `major_label_font_size`, `point_label_font_size`, `key_title_font_size`.

**Comment by DJ-kim** [comment]
What I meant was not the font size but the size of continuous color bar. Is it adjustable?

**Comment by dcjones** [comment]
Sorry, there's not a way to do this currently. Leave this issue open and I'll try to get around to adding a parameter.

**Comment by bjarthur** [comment]
relevant code is [here](https://github.com/GiovineItalia/Gadfly.jl/blob/master/src/guide.jl#L302) if anyone wants to take this on.  given that both [matlab](https://www.mathworks.com/help/matlab/ref/colorbar.html) and [python](https://matplotlib.org/examples/images_contours_and_fields/streamplot_demo_features.html) both have color bars which span the full height/width of the plot, i'd suggest making that the default here.  i'm not sure a parameter is really necessary.
